tasks/keylime-config.yml

```
@@ -0,0 +1,89 @@
+# SPDX-License-Identifier: MIT
+---
+- name: Make sure verifier snippet directory exists
+  ansible.builtin.file:
+    path: "{{ __keylime_server_verifier_snippet_dir }}"
+    state: directory
+    mode: '0500'
+    owner: "{{ __keylime_server_user }}"
+    group: "{{ __keylime_server_group }}"
+
+- name: Generate verifier ip/port configuration snippet
+  ansible.builtin.template:
+    src: "{{ __keylime_server_verifier_ip_config }}.j2"
+    dest: >-
+      {{ __keylime_server_verifier_snippet_dir }}/
```

```suggestion
      {{ __keylime_server_verifier_snippet_dir }}/\
```
otherwise this will expand as "/snippet/dir ipconfig" with a space in it - see https://linux-system-roles.github.io/documentation/howto/working-with-ansible-jinja-code-and-filters.html "Line Wrapping"

---

Fixed these as well, thanks.